### ISSUE TITLE
try to automate C code generation for Thrust (at least for phys formulae, not for algorithms)

### ISSUE THREAD

**Issue by slayoo** [atmos-cloud-sim-uj/PySDM#176]

**Comment by slayoo** [comment]
Well, for the record, this works:
```python
from PySDM.physics.formulae import radius
print(inspect.getsource(radius))
```
```
@numba.njit(**{**conf.JIT_FLAGS, **{'parallel': False}})
def radius(volume):
    return (volume * 3 / 4 / np.pi) ** (1 / 3)
```
```python
print(inspect.getsourcelines(radius))
```
```
(["@numba.njit(**{**conf.JIT_FLAGS, **{'parallel': False}})\n", 'def radius(volume):\n', '    return (volume * 3 / 4 / np.pi) ** (1 / 3)\n'], 148)
```
```python
print(inspect.signature(radius))
```
```
(volume)
```

@Kyetuur @tehAgitto @piotrbartman 
It might likely help in bringing some of the simpler physics formulae needed for condensation into Thrust?

**Comment by slayoo** [comment]
addressed in #484
